### DISCUSSION TITLE
"Left Sequence" and "Right Sequence"

### DISCUSSION THREAD

**Discussion by BinderHarald** [krasa/StringManipulation#244] [Ideas]
Hello,

If i want to create a Sequence with the example below it also sequences Fieldnames like FO_PREISLISTE_1.
Having an right-sequencing option which only renames the first numbers from the right might be very useful.

	private static final int FO_BRANCHE_NAME							= 41;
	private static final int FO_PREISE_RABATTE_RECHNUNGSEMPFAENGER	= 42;
	private static final int FO_FAHRER									= 43;
	private static final int FO_POSTEN_SALDO_EXTERN					= 44;
	private static final int FO_PREISLISTE_1							       = 45;
	private static final int FO_PREISLISTE_1_NAME						= 46;
	private static final int FO_PREISLISTE_2							        = 47;
	private static final int FO_PREISLISTE_2_NAME						= 48;
	private static final int FO_KOSTENSTELLEN_AUFTEILUNGSPROFIL			= 49;
	private static final int FO_KOSTENSTELLEN_AUFTEILUNGSPROFIL_NAME	= 50;
	
Similar Problem but from the left.	

		/* 005 */ addCollection(ci);
		/* 006 */ addCollection(BsSonderpreisVerkauf.class, BsSonderpreisVerkauf.FN_REF_KUNDE);
		/* 007 */ addCollection(BsKndPreisliste.class, BsKndPreisliste.FN_REF_KUNDE);
		/* 008 */ addCollection(BsVerkaufRabatt.class, BsVerkaufRabatt.FN_REF_KUNDE);
		/* 009 */ addCollection(BsProjekt.getConcreteClass(), BsProjekt.FN_REF_KUNDE);

best regards anfd thx,

Harald
Italy

**Comment by krasa** [comment]
I would multiselect `=`, then shift+end...
